Ginger/GingerCore/ALM/QtestCore.cs

```diff
@@ -0,0 +1,432 @@
+#region License
+/*
+Copyright © 2014-2019 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using Amdocs.Ginger.Common;
+using GingerCore.Activities;
+using GingerCore.ALM.QC;
+using GingerCore.ALM.Qtest;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using TDAPIOLELib;
+using Amdocs.Ginger.Repository;
+using Amdocs.Ginger.Common.InterfacesLib;
+using System.Linq;
+
+namespace GingerCore.ALM
+{
+    public class QtestCore : ALMCore
+    {
+        public QtestCore() { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)